saleor/graphql/product/tests/test_product.py

```diff
@@ -950,6 +950,21 @@ def test_products_query_with_filter_stocks(
     assert {node["node"]["id"] for node in products_data} == product_ids
 
 
+def test_query_products_query_with_filter_ids(
+    staff_api_client, product, query_products_with_filter
```

Comment:
If we execute request without permissions we can use `api_client` instead of `staff_api_client` .
```suggestion
    api_client, product, query_products_with_filter
```